sirepo/template/srw.py

```diff
@@ -40,6 +39,8 @@
 
 _CANVAS_MAX_SIZE = 65535
 
+_RSOPT_EXPORT_BASE_NAME = 'exportRsOpt'
```

Comment:
This is used in sim_data, and it seems like it should be defined there so it can be reused consistently throughout the code. I think EXPORT_RSOPT is probably a sufficient name for this purpose.